src/index.html

```diff
@@ -5,10 +5,12 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Colored blocks</title>
+    <title>Colored blocks123</title>
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Colored blocks</h1>
+    <div class="first block"></div>
+    <div class="second block"></div>
+    <div class="third block"></div>
```

Comment:
if you have looked through the [checklist](https://github.com/mate-academy/layout_colored-blocks/blob/master/checklist.md), you must have noticed recommendation about class names

They should be descriptive and show how the element with this class interact with others.

In your case they could be for example `outer-block`, `mid-block`, `inner-block`